### DISCUSSION TITLE
Missing translation of Netflix quality check

### DISCUSSION THREAD

**Discussion by potplayer-fanpack** [SubtitleEdit/subtitleedit#7500] [Show and tell]
Hi, nikse 

Can you fill in the missing translation of Netflix quality check?

![Netflix-quality-check](https://github.com/SubtitleEdit/subtitleedit/assets/117507909/39ac8ebb-6872-4619-ba28-65af2ede1ec1)

**Comment by niksedk** [comment]
I was hoping to re-work this at some point... but have not found the time/inspiration.
